nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/tools/IntegralTool.java

```diff
@@ -121,4 +144,17 @@ public void splitRegion() {
         }
 
     }
+
+    public void deleteRegion() {
```

Comment:
The integral at the top is deleted, the multiplet is still visible though, this seems to be what the old RegionTool was doing as well...

Reply:
now delete any peaks within the deleted region